web/crux-ui/src/components/shared/json-editor.tsx

```diff
@@ -31,64 +58,36 @@ const _JsonEditor = <T,>(props: JsonEditorProps<T>) => {
 
     try {
       const json = JSON.parse(text)
-      props.onChange?.call(null, json)
+      propOnChange?.call(null, json)
     } catch (e) {
       const err = e as Error
-      props.onParseError?.call(null, err)
+      onParseError?.call(null, err)
     }
   }
 
   return (
     <div
-      className={clsx(
-        'text-bright bg-gray-900 rounded-md ring-2 ring-light-grey border-dark caret-white',
-        props.className,
-      )}
+      className={clsx('text-bright bg-gray-900 rounded-md ring-2 ring-light-grey border-dark caret-white', className)}
     >
       <Editor
         textareaClassName="outline-none"
-        disabled={props.disabled}
+        disabled={disabled}
         padding={2}
         tabSize={2}
         insertSpaces
         value={state}
         onValueChange={onChange}
-        highlight={value => highlight(value, languages['json'], 'json')}
+        highlight={newValue => highlight(newValue, languages.json, 'json')}
       />
     </div>
   )
 }
 
-export default _JsonEditor
+export default JsonEditor
```

Comment:
Rename it to JsonEditorInternal or something, cause we're using dynamic module on it